timer.js

```diff
@@ -36,18 +36,19 @@ function play() {
     }
     else
         startTimer();
+    timerRunning = True;
```

Comment:
maybe add four spaces so it aligns with `startTimer()`? 🙂

Reply:
ohh good catch